### ISSUE TITLE
Error: Cannot find module 'cra-template-redux-typescript'

### ISSUE THREAD

**Issue by dtgoitia** [reduxjs/cra-template-redux-typescript#5]
Hi guys! I was giving a try to this template but it failed on installation:

```shell
13:28:48 ❯ npx create-react-app . --template redux-typescript
npx: installed 99 in 6.706s

Creating a new React app in /Users/david.torralba/projects/tempus/tempus-ui.

Installing packages. This might take a couple of minutes.
Installing react, react-dom, and react-scripts with cra-template-redux-typescript...

yarn add v1.21.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "react-scripts > eslint-config-react-app@5.2.0" has incorrect peer dependency "eslint-plugin-flowtype@3.x".
warning "react-scripts > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 9 new dependencies.
info Direct dependencies
├─ cra-template-redux-typescript@0.0.1
├─ react-dom@16.12.0
├─ react-scripts@3.4.0
└─ react@16.12.0
info All dependencies
├─ cra-template-redux-typescript@0.0.1
├─ is-docker@2.0.0
├─ open@7.0.2
├─ react-dev-utils@10.2.0
├─ react-dom@16.12.0
├─ react-error-overlay@6.0.6
├─ react-scripts@3.4.0
├─ react@16.12.0
└─ scheduler@0.18.0
✨  Done in 19.01s.
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'cra-template-redux-typescript'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.resolve (internal/modules/cjs/helpers.js:32:19)
    at module.exports (/Users/david.torralba/projects/tempus/tempus-ui/node_modules/react-scripts/scripts/init.js:110:13)
    at [eval]:3:14
    at Script.runInThisContext (vm.js:96:20)
    at Object.runInThisContext (vm.js:303:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at evalScript (internal/bootstrap/node.js:587:27)
    at startup (internal/bootstrap/node.js:265:9)

Aborting installation.
  node  has failed.

Deleting generated file... node_modules
Deleting generated file... package.json
Deleting generated file... yarn.lock
Done.
```

NVM: `1.0.1`
Node: `v13.9.0`
NPM: `6.13.7`

Any suggestions?

**EDIT**:
I've tried other templates and work fine:

```
npx create-react-app . --template redux-typescript  # fail
npx create-react-app . --template typescript  # OK
npx create-react-app . --template redux  # OK
```

**Comment by nickmccurdy** [comment]
This package is still in development, the `0.0.1` version doesn't contain a working template yet. It will likely be published after #3 is complete.

**Comment by dtgoitia** [comment]
Thanks @nickmccurdy! I just couldn't wait to get my hands on it 😄
